openwisp_ipam/api/__init__.py

```diff
@@ -1,37 +0,0 @@
-from django.urls import path
```

Comment:
This does not seem to be the right place for URLs. I had to look for them, hence moved them to a dedicated file.